UI/profile.html

```diff
@@ -0,0 +1,117 @@
+						<h1>APC</h1>
+                        <span>Raising future leaders</span>
+                        <button class="vote-button">Vote</button>
+					</div>
+				</div>
+				<div class="vote-card-area">
+					<div class="vote-card" id="img2">
+						<h1>PDP</h1>
+                        <span>Raising future leaders</span>
+                        <button class="vote-button">Vote</button>
+					</div>
+                </div>
+                <div class="vote-card-area">
+                        <div class="vote-card" id="img3">
+                            <h1>ACC</h1>
+                            <span>Raising future leaders</span>
+                            <button class="vote-button">Vote</button>
+                        </div>
+                    </div>
+                    <div class="vote-card-area">
+                        <div class="vote-card" id="img4">
+                            <h1>CZZ</h1>
+                            <span>Raising future leaders</span>
+                            <button class="vote-button">Vote</button>
+                        </div>
+                    </div>
+                    <div class="vote-card-area" >
+                            <div class="vote-card" id="img5">
+                                <h1>APP</h1>
+                                <span>Raising future leaders</span>
+                                <button class="vote-button">Vote</button>
+                            </div>
+                        </div>
+                        <div class="vote-card-area">
+                            <div class="vote-card" id="img6">
+                                <h1>APC</h1>
+                                <span>Raising future leaders</span>
+                                <button class="vote-button">Vote</button>
+                            </div>
+                        </div>
+                        <div class="vote-card-area">
+                                <div class="vote-card" id="img7">
+                                    <h1>LCA</h1>
+                                    <span>Raising future leaders</span>
+                                    <button class="vote-button">Vote</button>
+                                </div>
+                            </div>
+
+                            <div class="vote-card-area">
+                                    <div class="vote-card" id="img8">
+                                        <h1>ALC</h1>
+                                        <span>Raising future leaders</span>
+                                        <button class="vote-button">Vote</button>
+                                    </div>
+                                </div>
+                   
+				
+					</div>
+				</div>
+<!--                 <footer class="politico-footer">
+                        <p>politico. &copy; 2019; All pictures are gotten from google and are real political parties logo </p>
+                    </footer> -->
+	</body>
+</html>
```

Comment:
A new line here, please. Also, kindly format your html, so it looks well. Thanks